openlibrary/plugins/openlibrary/borrow_home.py

```diff
@@ -17,14 +17,14 @@
 from openlibrary.plugins.worksearch.subjects import SubjectEngine
 
 
-class borrow(delegate.page):
+class borrow(delegate.page):  # TODO: Why is this function defined twice?
     path = "/borrow"
 
     def GET(self):
         raise web.seeother('/subjects/in_library#ebooks=true')
 
 
-class borrow(delegate.page):
+class borrow(delegate.page):  # type: ignore[no-redef]
```

Comment:
@cclauss, do you think that should be named something like `borrow_json`?  Some of our classes in https://github.com/internetarchive/openlibrary/blob/master/openlibrary/plugins/openlibrary/lists.py follow this pattern.